include/oneapi/dpl/internal/distributed_ranges_impl/concepts/concepts.hpp

```diff
@@ -0,0 +1,81 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
The current oneDPL practice is to use include guards, not `#pragma once`.

The pragma is non-standard but de-facto highly portable (https://en.wikipedia.org/wiki/Pragma_once#Portability). Nevertheless, none of the three major standard library implementations uses this pragma in their public headers.

I can probably leave with it for now, but eventually I prefer it to be replaced with include guards - unless we take a thoughtful decision to prefer the pragma to include guards.

Reply:
applied, merged, resolving